src/main/java/com/minecolonies/util/ServerUtils.java

```diff
@@ -79,6 +80,27 @@ public static EntityPlayer getPlayerFromUUID(@NotNull World world, @NotNull UUID
         return players;
     }
 
+    //todo: document
+    @NotNull
+    public static List<EntityPlayer> getPlayersFromPermPlayer(@NotNull List<Permissions.Player> players)

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)